lib/Conversion/TritonGPUToLLVM/ReduceOpToLLVM.cpp

```diff
@@ -170,9 +170,23 @@ struct ReduceOpConversion
     if (success)
       return;
     for (unsigned N = numLaneToReduce / 2; N > 0; N >>= 1) {
+      unsigned shuffleIdx = N;
+      auto srcTys = op.getInputTypes();
+      auto inputTy = srcTys[0].cast<RankedTensorType>();
+      auto inMfma =
+          inputTy.getEncoding().dyn_cast<triton::gpu::AMDMfmaEncodingAttr>();
+      if (inMfma && inMfma.getIsTransposed() && op.getAxis() == 1) {
```

Comment:
I thought we discussed this previously that we should be able to rely on `getThreadOffsetOnReductionAxis`

Reply:
Nevermind, I tried running without this and it does seem to work.